migrations/versions/0360_new_user_bcast_permissions.py

```diff
@@ -0,0 +1,32 @@
+"""
+
+Revision ID: 0360_new_user_bcast_permissions
+Revises: 0359_more_permissions
+Create Date: 2021-06-30 11:42:32.780734
+
+"""
+from alembic import op
+
+revision = '0360_new_user_bcast_permissions'
+down_revision = '0359_more_permissions'
+
+
+def upgrade():
+    """
+    Delete all permissions for broadcast service users and invited pending users, apart from 'view_activity'
+    which they always have.
+    """
+    op.execute(
+        "DELETE FROM permissions WHERE permission != 'view_activity' "
+        "and service_id in (select id from services where organisation_id = '38e4bf69-93b0-445d-acee-53ea53fe02df')"
+    )
+    op.execute(
+        "UPDATE invited_users SET permissions = 'view_activity' WHERE status = 'pending' "
```

Comment:
This will give invited users the view permission even if they didn't have it before. I don't think that's a big deal, though: I'm not aware of any reason why we need to hide anything from particular broadcast service users.

Reply:
The `view_activity` permission is [always added](https://github.com/alphagov/notifications-admin/blob/256c840b46d4095edbd0af105e5c1a340242575d/app/main/forms.py#L1042) to the list of permissions when either the `BroadcastPermissionsForm` or the `BroadcastInviteUserForm` is submitted, so this keeping the same behaviour.

Reply:
Ahh, that's good to know, thanks.

I never noticed they were absent from the form before.